### DISCUSSION TITLE
Generate Getter and Setter

### DISCUSSION THREAD

**Discussion by Srkless** [mfussenegger/nvim-jdtls#693] [Q&A] [answered]
Is it possible to change prefix in code generator for setter and getter? My atribute is for example:
private String m_name;

Can I autoGenerate getter like getName, and not getM_name?

**Comment by mfussenegger** [comment, marked as answer]
As far as I'm aware there is currently no option to customize the getters. You'd have to ask in the eclipse.jdt.ls project for a new configuration option. Similar to the existing `CodeGenerationOption` in https://github.com/eclipse-jdtls/eclipse.jdt.ls/wiki/Running-the-JAVA-LS-server-from-the-command-line#initialize-request
